config/settings.yml

```diff
@@ -3342,6 +3342,32 @@ theaters:
           n: NORTH
           e: EAST
           s: SOUTH
+      roosevelt:
+        name: CVN-71 T.Roosevelt
+        tacan: 74X
+        tower: 127.5
+        approach: 264.0
+        clearance: 265.0
+        ground: ICLS 12
+        departures:
+          cat: CAT
+        recoveries:
+          case1: CASE I
+          case2: CASE II
+          case3: CASE III
+      lincoln:
+        name: CVN-72 A.Lincoln
+        tacan: 75X
+        tower: 128.5
+        approach: 264.0
+        clearance: 265.0
+        ground: ICLS 13
+        departures:
+          cat: CAT
+        recoveries:
+          case1: CASE I
+          case2: CASE II
+          case3: CASE III          
```

Comment:
This is special to your mission/template. I cannot merge this in the master. Can you exclude this from the PR please?

Reply:
Sure